apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -80,6 +81,67 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
 
     const endUserSession = useEndUserSession();
 
+    interface iconProps {
+        class : string;
+        color : SemanticCOLORS;
+    }
+
+    const getIconProps = (id : string) : iconProps =>{
+                
+        const iconProps = {
+            class : "circle",
+            color : "grey" as SemanticCOLORS
+        };
+
+        const lowerCaseLetters = /[a-z]/g;
+        const upperCaseLetters = /[A-Z]/g;
+        const numbers = /[0-9]/g;
+        const chars = /[!@#$%^&*~]/g;
+
+        if ( id === "password-validation-length" ){
+            if (password === "") {
+                iconProps.class = iconProps.class + " thin"
+            } else if ( password.length > 8 ) {
+                iconProps.class = "check " + iconProps.class
+                iconProps.color = "green" as SemanticCOLORS;
+            } else {
+                iconProps.class = "remove " + iconProps.class
+                iconProps.color = "red" as SemanticCOLORS;
+            }
+        } else if ( id === "password-validation-case" ) {
```

Comment:
No need to keep spaces inside before/after the parenthesis.
Let's use ESLint plugin to format the code.

```suggestion
        } else if (id === "password-validation-case") {
```